src/ExpressiveInstaller/Resources/src/ExpressiveAuraConfig.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace App;
+
+use ArrayObject;
+use Aura\Di\Container;
+use Aura\Di\ContainerConfigInterface;
+
+/**
+ * Configuration for the Aura.Di container.
+ *
+ * This class provides functionality for the following service types:
+ *
+ * - Aliases
+ * - Delegators
+ * - Factories
+ * - Invokable classes
+ * - Services (known instances)
+ */
+class ExpressiveAuraConfig implements ContainerConfigInterface
```

Comment:
That's what [service-provider](https://github.com/container-interop/service-provider) is tackling. :)

The idea then will be that container providers would consume the service-provider interface in order to wire services to the container. The way the interface is written, it can be either a straight factory, or act as a delegator, which solves the majority of problems (invokables and aliases may be accomplished using a combination of these).

My plan is to bring some our ideas and findings to that project/working group, instead of creating a project of our own.